packages/studio/src/store/slices/pages/createPageSlice.ts

```diff
@@ -187,10 +215,15 @@ export const createPageSlice: SliceCreator<PageSlice> = (set, get) => {
         activeComponentUUID,
         getActivePageState,
         addSelectedComponentUUID,
+        clearSelectedComponents,
       } = get();
       const activePageState = getActivePageState();
       const componentTree = activePageState?.componentTree;
-      if (selectedUUID === activeComponentUUID) return;
+      if (selectedUUID === activeComponentUUID) {
+        clearSelectedComponents();
+        addSelectedComponentUUID(selectedUUID);
+        return;
```

Comment:
can we add/update some unit tests, particularly for store actions that were added/changed?

Reply:
Added extra tests for updateSelectedComponent and getFirstSelectedComponentUUID